.circleci/config.yml

```diff
@@ -126,69 +126,6 @@ jobs:
           no_output_timeout: 20m
           command: inv -e linter.go
```

Comment:
and why not this one as well? It seems like it's already launched in `.linux_lint` so we can probably deactivate it, wdyt?

Reply:
Good point

Reply:
But I chose to remove the circleCI jobs in a different PR, so I will however put the circleCI code back.